less/shared.less

```diff
@@ -514,7 +515,7 @@ select.parsley-error:focus {
 
 @media screen and (max-width: 480px) {
   .container {
-    padding: 10px;
+    border-radius:0px;
```

Comment:
You want to add a space between `border-radius:` and `0;`

Also, general practice in mofo is, `0px;` isn't necessary, and should just be `0;`